src/main/java/com/pscheduler/util/CourseBuilderFactory.java

```diff
@@ -18,6 +18,7 @@
     private List<Meeting> meetings;
     private String exam;
     private int term;
+    private double gpa;
```

Comment:
Remove changes to this file and the parser. We'll have a separate parser for the GPA courses

Reply:
Done